clients/discord-bot/config/default.json

```diff
@@ -14,15 +14,15 @@
   "gql": {
     "url": "http://localhost:3001/graphql"
   },
-  "logger": {
-    "levels": true
-  },
-  "minio": {
-    "endPoint": "",
+  "s3": {
```

Comment:
Do we want these in here?

Reply:
[b77d469](https://github.com/SprocketBot/sprocket/pull/457/commits/b77d4698364dc08bc2ce5204d8394fe0b0b33e3a)